core/models/instance_history.py

```diff
@@ -16,6 +16,20 @@ class InstanceStatus(models.Model):
     """
     Used to enumerate the types of actions
     (I.e. Stopped, Suspended, Active, Deleted)
+
+    FIXME: (Idea) -- adding a new field, is_final_state
+    Example of 'is_final_state' status for Openstack:
+        - active
+          suspended
+          shutoff
+          error
+          deleted
+          unknown
+    Example of 'is_final_state = False' status for Openstack:
+        - networking
+          deploying
+          deploy_error
+
```

Comment:
Would having a "pending" state to show a resource in transition (i.e. - waiting on an async action within the task queue) be something to consider? Or, would that be a "faux" state for AtmoAPI use only and this is only OpenStack states?